src/main/resources/db/local/V99_0__draft_referrals.sql

```diff
@@ -1,4 +1,3 @@
 insert into referral (id, created, completion_deadline)
-values ('F7B4A237-94CC-448F-A2C0-CDFFF17B58B9', '2020-12-07 18:02:01.599803+00', '2021-02-14'),
-       ('71E6EAF6-5899-4A6B-9651-1AA14B3843A3', '2020-12-07 20:45:21.986389+00', '2021-03-01');
-
+values ('ac386c25-52c8-41fa-9213-fcf42e24b0b5', '2020-12-07 18:02:01.599803+00', '2021-02-14'),
+       ('dfb64747-f658-40e0-a827-87b4b0bdcfed', '2020-12-07 20:45:21.986389+00', '2021-03-01');
```

Comment:
cc @green-digger -- I may need to remove `local` from the codeowners for data engineering 🤔